alf/algorithms/sac_algorithm.py

```diff
@@ -447,6 +450,15 @@ def _filter(x):
             tau=target_update_tau,
             period=target_update_period)
 
+        self._periodic_reset = common.PeriodicReset(
+            models=_filter([
+                self._actor_network, self._critic_networks,
+                self._target_critic_networks, repr_alg, target_repr_alg,
+                self._log_alpha
+            ]),
+            post_processings=[self._update_target],
```

Comment:
Maybe we can skip the ``TargetUpdater`` here given that ``self._target_critic_networks`` are also reset?

Reply:
> Maybe we can skip the `TargetUpdater` here given that `self._target_critic_networks` are also reset?

We need to keep the post_preocessing since we need to re-copy the param values from critic network to target whenever a reset is done.

Reply:
Yes, I think the updated version is faithful to the original paper.